controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const {getTopicInfo} = require('../models/models');
+
+exports.getTopics = (req,res) => {
+    getTopicInfo().then((rows) => 
```

Comment:
We could declare this parameter as 'topics' so that later when we send it we are just sending {topics} (same key and value)